src/Microsoft.AspNet.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -312,11 +312,11 @@ private ViewLocationCacheResult LocatePageFromPath(string pageName, bool isParti
             }
 
             var cacheKey = new ViewLocationCacheKey(
-                expanderContext.ViewName,
-                expanderContext.ControllerName,
-                expanderContext.ViewName,
-                expanderContext.IsPartial,
-                expanderValues);
+                viewName: expanderContext.ViewName,
+                controllerName: expanderContext.ControllerName,
+                areaName: expanderContext.AreaName,
+                isPartial: expanderContext.IsPartial,
+                values: expanderValues);
```

Comment:
Why bother naming the parameters? Would have helped when the code was originally written but looks overly verbose now.